doc/mgr/dashboard.rst

```diff
@@ -126,15 +126,26 @@ For example, a key pair can be generated with a command similar to::
 The ``dashboard.crt`` file should then be signed by a CA. Once that is done, you
 can enable it for all Ceph manager instances by running the following commands::
 
-  $ ceph config-key set mgr mgr/dashboard/crt -i dashboard.crt
-  $ ceph config-key set mgr mgr/dashboard/key -i dashboard.key
+  $ ceph config set mgr mgr/dashboard/crt -i dashboard.crt
```

Comment:
I should have been more specific in my earlier comment -- the setting for ssl needed to be "config set", but these should still be "config-key set" (they're "store" entries rather than config entries http://docs.ceph.com/docs/master/mgr/plugins/#kv-store)